packages/traveling-fastlane/actions/list_devices.rb

```diff
@@ -1,17 +1,44 @@
 require 'spaceship'
 require 'json'
 require 'base64'
+require 'optparse'
 require_relative 'funcs'
 
+options = {}
+OptionParser.new do |opts|
```

Comment:
Same as in the other PR, if you would like to use --named-parameters please convert all of the existing positional parameters to names ones.